changes/5569.changed

```diff
@@ -0,0 +1 @@
+Changed logic in determining if a relationship side is required to ensure object instance exists in the filter.
```

Comment:
1) Should this be `.fixed` since it's addressing a bug?
2) Can we make this any more clear about specifically what it's fixing?